exporter/README.md

```diff
@@ -37,13 +37,48 @@ exporters:
 ```
 
 ## <a name="logging"></a>Logging
-TODO: document settings 
+TODO: document settings
 
 ## <a name="opencensus"></a>OpenCensus
-TODO: document settings 
+Exports traces and/or metrics to another OTel-Svc endpoint via gRPC.
+
+### <a name="opencensus-configuration"></a>Configuration
+
+* `endpoint`: target to which the exporter is going to send traces or metrics,
+using the gRPC protocol. The valid syntax is described at
+https://github.com/grpc/grpc/blob/master/doc/naming.md
+
+* `compression`: compression key for supported compression types within
+opencensus collector. Currently the only supported mode is `gzip`.
+
+* `headers`: the headers associated with gRPC requests.
+
+* `num-workers`: number of workers that send the gRPC requests.
+
+* `secure`: whether to enable client transport security for the exporter's gRPC
+connection. See [grpc.WithInsecure()](https://godoc.org/google.golang.org/grpc#WithInsecure).
+
+* `cert-pem-file`: certificate file for TLS credentials of gRPC client. Should
+only be used if `secure` is set to true.
+
+* `reconnection-delay`: time period between each reconnection performed by the
+exporter.
+
+* `keepalive`: keepalive parameters for client gRPC. See
+[grpc.WithKeepaliveParams()](https://godoc.org/google.golang.org/grpc#WithKeepaliveParams).
+
+Example:
+
+```yaml
+exporters:
+  opencensus:
+    endpoint: 127.0.0.1:14250
+    reconnection-delay: 60s
+    secure: false
```

Comment:
It would be nice if the example could contains `pipeline` which refers to the defined exporter so that uses know how to use the exporters in pipeline, e.g.,

```
pipelines:
  metrics:
    receivers: [opencensus]
    exporters: [opencensus, logging]
```

Reply:
This readme is mostly focused on exporter section. We can add a similar example to pipeline readme or example config.

Reply:
Sounds good. Thank you.

Reply:
The example config will be valid and will exemplify how to reference an exporter(/processor/receiver).
Also, the pipeline documentation will reiterate how to specify valid pipelines.